Update how-to-write-a-parser.md to include advice for CVSS data (#4685)
DefectDojo/django-DefectDojo: docs/content/en/contributing/how-to-write-a-parser.md

       finding.cve = data["mykey"]
```

> ### Do not parser CVSS by hand (vector, score or severity)

Reviewer: ```suggestion
### Do not parse CVSS by hand (vector, score or severity)
```

> Data can have `CVSS` vectors or scores. Don't try to right again your own CVSS score algorythm.

Reviewer: ```suggestion
Data can have `CVSS` vectors or scores. Don't try to write your own CVSS score algorithm.
```

> For parser, we rely in module `cvss`.

Reviewer: ```suggestion
For parser, we rely on module `cvss`.
```
